wrapanapi/entities/volume.py

```diff
@@ -0,0 +1,85 @@
+"""
+wrapanapi.entities.volume
+
+Volumes
+"""
+from __future__ import absolute_import
```

Comment:
We're python 3 only now, shouldn't have to use these two here. I might be missing something on absolute_import though, was there a specific reason you used it here?

Reply:
No, I just copied that from different entity.

Reply:
@mshriver If needed I can just remove it.